source/color_raw_to_rgb.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<h2>Prepare the raw file through rawpy</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>